.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
this is your local vscode workspace settings file, not something that is part of the project, please add the .vscode folder to the gitignore file.

Reply:
I intentionally added these option to the vscode settings of the project to support development with the right settings within vscode (i.e. pytest instead of unittest & autopep8 as a formatter). From my perspective this is comparable to tox or other dotfiles/helpers for the software development due to its standardization of the process. 
However, if you disagree on having these information within the repo I am also happy to remove it.

Reply:
As you wish.

Reply:
if you add a file that was previously tracked to gitignore, you need to remove it from git with `git rm --cached <file>` or `git rm -r --cached <folder>`. Quickly did it, just pull the latest changes ;)